api/app/Services/Zoom/ZoomService.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Services\Zoom;
+
+use App\Repositories\User\UserRepository;
+use Illuminate\Support\Facades\Http;
+
+final class ZoomService
+{
+    private const ZOOM_MEETING_API_ENDPOINT = 'https://api.zoom.us/v2/users/me/meetings';
+    private const ZOOM_GET_USER_API_ENDPOINT = 'https://api.zoom.us/v2/users/me';
+    private const ZOOM_OAUTH_TOKEN_POST_REQUEST_URL = 'https://zoom.us/oauth/token';
+
+    private $userRepository;
+
+    public function __construct(UserRepository $userRepository)
```

Comment:
The property type is missing. Also, you should use `UserRepositoryInterface` instead.